### ISSUE TITLE
SunPy has two different functions that calculate the Sun's angular radius

### ISSUE THREAD

**Issue by wafels** [sunpy/sunpy#4516]
### Description

SunPy has two different functions that calculate the Sun's angular radius:

https://github.com/sunpy/sunpy/blob/master/sunpy/map/maputils.py#L84
https://github.com/sunpy/sunpy/blob/master/sunpy/coordinates/sun.py#L62

### Expected vs Actual behavior

I expect that there should be only one function in SunPy to calculate the Sun's angular radius.

### System Details

This is in the currently released version of SunPy.

**Comment by ayshih** [comment]
For completeness, I'll note that this line doesn't call either function:
https://github.com/sunpy/sunpy/blob/2b8ee1b25ee16199b92e9d396675e51e457b1368/sunpy/map/header_helper.py#L227

**Comment by ayshih** [comment]
As an aside, the arcsin/arctan disagreement is fixed by #4518

**Comment by BassCoder2808** [comment]
So basically  we want to have only one function which will calculate the expected as well as the actual Sun's angular radius. 
But these functions are in 2 different folders. So which function do you want to retail ?
And do we have to create a new function in all or just modify it.
I would like to contribute on this issue.

**Comment by nabobalis** [comment]
Hi @BassCoder2808, yes you are correct but a PR has been just opened addressing this. 

We would strongly suggest having a look at some other issues.

**Comment by ayshih** [comment]
#4524 fixes all of the underlying code issues, but there still is an API issue of whether we want two functions in two different places that both return the Sun's angular radius, even though the inputs are quite different.  I'll point out that the function in `map/maputils.py` is a bit odd since it doesn't actually involve a Map in anyway, so maybe it should be moved to `coordinates/utils.py` regardless?

The residual issue is no longer a "good first issue", so I am removing the label.

**Comment by jeffreypaul15** [comment]
Has this issue been resolved? Or as @ayshih stated above should the code from `map/maputils.py` be moved into `coordinates/utils.py`?

**Comment by nabobalis** [comment]
I don't believe the move was made in a pull request. We should probably have go.
